### PR TITLE
Setup Java Logging Framework #17

### PR DESCRIPTION
**Pull request by n-machiavelli**
Add logback files for dev and prod.
logback uses sl4j library.

### CODE REVIEW

**Comment by Code4SocialGood** [comment]
1. Should we not check in `logFile.2017-04-07.log` to the repository? - DELETED
2. Where will be the log file dumped? Could we create a directory just for log files? - the logback file will create logs directory now.

File: src/main/java/org/c4sg/entity/User.java [hunk 1]
```diff
@@ -90,6 +90,9 @@
 		
 	@Column(name = "updated_time")
 	private Date updatedTime;
+	
+    @Column(name = "title")
+    private String title;
```
Comment (felipealvesgnu) [inline src/main/java/org/c4sg/entity/User.java]: Pay attention on indentation

**Review by felipealvesgnu — CHANGES_REQUESTED** [review]

File: src/main/java/org/c4sg/entity/User.java (hunk 1 above)
Comment (n-machiavelli) [reply src/main/java/org/c4sg/entity/User.java]: Oh I must have used spaces instead of a tab. Thanks.

Comment (felipealvesgnu) [reply src/main/java/org/c4sg/entity/User.java]: solve it and push the code again

**Comment by felipealvesgnu** [comment]
@n-machiavelli Can you answer @Code4SocialGood 's questions ?

**Comment by n-machiavelli** [comment]
Should we not check in logFile.2017-04-07.log to the repository? - DELETED as per Slack convo
Where will be the log file dumped? Could we create a directory just for log files? - the logback file will create logs directory now as per slack convo

**Comment by Code4SocialGood** [comment]
How to get a log file generated? I called getActiveUser API fromo http://localhost:8080/swagger-ui.html#!/user/getActiveUsersUsingGET, which calls  LOGGER.debug("**************All**************"); But I didn't get a log file generated locally.

image

**Comment by n-machiavelli** [comment]
This has been resolved. Note that you won't have "log.debug" in the log file as prod is set to INFO level. but if you have log.info for testing you will find it:
image
image

**Comment by Code4SocialGood** [comment]
1. Are we using log4j2 or logback? The naming of 3 logging config files is confusing: log4j2.xml, logback-dev.xml, logback-prod.xml
2. Local Development Environment: After starting Spring Boot, a log file is generated: \logs\logFile.2017-04-13.log. Please change the config so that all the logging are sent to console. No log file for local dev.
3. Test server environment and Prod environment: Log level is DEBUG for test server and INFO for prod. Logging is sent to file appender. What do I need to change so that test/prod server config will be used?

**Comment by n-machiavelli** [comment]
1. Are we using log4j2 or logback? We are using logback. The log4j2 file was there before and I have just removed it and retested.
2. I have fixed this in application-dev.properties. No log file for local dev.
3. This is setup by setting the appropriate logback file in the last line of application-dev.properties and application-prod.properties

image

image

**Review by felipealvesgnu — APPROVED** [review]

**Pull request merged**